pkg/common-controller/snapshot_controller.go

```diff
@@ -663,6 +754,7 @@ func (ctrl *csiSnapshotCommonController) createSnapshotContent(snapshot *crdv1.V
 	// Update content in the cache store
 	_, err = ctrl.storeContentUpdate(updateContent)
 	if err != nil {
+		ctrl.metricsManager.RecordMetrics(createOperation, NewSnapshotOperationStatus(snapshotStatusTypeControllerError))
```

Comment:
I wonder if we need to record an error here.  We don't even return err.

Reply:
Added return err and metric emit is handled by defer